src/Bridge/Doctrine/Orm/Filter/DateFilter.php

```diff
@@ -94,7 +94,7 @@ protected function filterProperty(string $property, $values, QueryBuilder $query
         }
 
         $nullManagement = $this->properties[$property] ?? null;
-        $type = $this->getDoctrineFieldType($property, $resourceClass);
+        $type = (string) $this->getDoctrineFieldType($property, $resourceClass);
```

Comment:
would be great if we could directly return `string` in the AbstractFilter method instead

Reply:
done, thanks